doc/benchmarking.md

```diff
@@ -11,7 +11,7 @@ Running
 For benchmarks purposes you only need to compile `bitcoin_bench`. Beware of configuring without `--enable-debug` as this would impact
 benchmarking by unlatching log printers and lock analysis.
 
-    make -C src bench_bitcoin
+    make -C src bitcoin_bench
```

Comment:
Does `make -C src bench` work as well?

Reply:
Oh, I guess that also *runs* the bench

Reply:
Yes -- the target `bench` both builds and executes the benchmarks, `bitcoin_bench` only builds them.